### PR TITLE
Added helper function

### PR DESCRIPTION
**Pull request by SamarKaranch**
The helper function gets all the code file's embeddings from a repo

### CODE REVIEW

File: backend/app/api/routes.py
Comment (libioco) [inline backend/app/api/routes.py:1]: Looks like these are just for debugging - are these lines necessary for this function?

**Review by libioco — COMMENTED** [review]
Looks good, just needs one comment addressed

File: backend/app/api/routes.py
Comment (libioco) [reply backend/app/api/routes.py:1]: Unless these are for logs?

File: backend/app/api/routes.py [hunk 1]
```diff
@@ -474,7 +474,10 @@ def send_initialized_data_to_db(repo_info, code_files):
                 upsert=True
             )
 
+        repo_embeddings = db.get_repo_files_embeddings(repo_id)
```
Comment (tachille4) [inline backend/app/api/routes.py:477]: This function call should be moved to the /report endpoint after embeddings are recomputed (i.e., line 126)

Where it is currently may become outdated if SHAs are different between bug reports.

**Review by tachille4 — APPROVED** [review]

**Pull request merged**